service/geopmdpy_test/TestPlatformService.py

```diff
@@ -497,7 +498,7 @@ def test_start_batch_write_blocked(self):
         self.open_mock_session('other', other_pid)
         with mock.patch('geopmdpy.pio.write_control', return_value=[]) as mock_write_control, \
              mock.patch('geopmdpy.pio.save_control', return_value=[]), \
-             mock.patch('os.getsid', return_value=other_pid) as mock_getsid:
+             mock.patch('os.getsid', return_value=client_sid) as mock_getsid:
```

Comment:
Are any of the changes here expressly related to the `geopm_context` removal?

Reply:
This is to support the change to check that the PID holding the write lock is still active.